src/MembershipsSDKAdapter.js

```diff
@@ -31,23 +31,43 @@ const MAX_MEMBERSHIPS = 1000;
  * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/MembershipsAdapter.js#L6}
  */
 
+/**
+ * Sort the members alphabeticaly but puts the current user first
+ *
+ * @param {Array} members List of members from meeting
+ * @returns {Array} Sorted list of users in a meeting
+ */
+function sortMeetingMembers(members) {
+  return members.sort((member1, member2) => {
+    if (member1.isSelf) return -1; // current user comes first
+    if (member2.isSelf) return 1;
+    if (!member2.name) return -1; // empty names come last
+    if (!member1.name) return 1;
+
+    return member1.name.localeCompare(member2.name); // alphabetical order
+  });
+}
+
 /**
  * Gets the active members in a meeting
  *
  * @private
- * @param {object} members Members object from meeting, keyed by ID
- * @returns {Array} List of active users in a meeting
+ * @param {object} sdkMembers Members object from meeting
+ * @returns {Array} List of users in a meeting
```

Comment:
you can use the adapter `Member` type here:
```js
* @returns {Member[]} List of meeting members
```

and please remove the "active" word from the function description on line 52, too.